app/templates/summary.html

```diff
@@ -30,51 +30,60 @@ <h2 class="header__subtitle mars">Please check carefully before submission.</h2>
 
         <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">{{item.question | safe}}</dt>
         <dd class="summary__answer summary__answer--{{item.type}}">
+          {% for sub_item in item.sub_items %}
 
-          <div class="summary__answer-text" id="{{id}}-answer" data-qa="answer-{{outer_loop_index}}-{{inner_loop_index}} answer-type-{{item.type}}">
+            {% set item_id = id ~ "-" ~ loop.index0 %}
+            {% if item.sub_items|length > 1 %}
+              <dt class="summary__question summary__answer__title--{{sub_item.type}}" id="{{item_id}}">{{sub_item.answer_title | safe}}</dt>
+              <dd class="summary__answer summary__answer--{{item.type}}">
```

Comment:
Just a heads up: this isn't valid html as the `dt/dd` pair needs to be directly nested inside a `dl`. Also the `dd` tag isn't closed anywhere.

This bit of template no longer makes sense semantically anyway so I'm going to have to refactor the whole thing.